spec/models/bulkrax/csv_entry_spec.rb

```diff
@@ -90,29 +90,63 @@ module Bulkrax
+            'single_object_first_name' => 'Fake',
+            'single_object_last_name' => 'Fakerson',
+            'single_object_position' => 'Leader, Jester, Queen',
+            'single_object_language' => 'english'
           )
         end
 
         it 'succeeds' do
           metadata = subject.build_metadata
-          expect(metadata['creator']['creator_first_name']).to eq('Fake')
-          expect(metadata['creator']['creator_last_name']).to eq('Fakerson')
-          expect(metadata['creator']['creator_position']).to include('Leader', 'Jester', 'Queen')
-          expect(metadata['creator']['creator_language']).to eq('English')
+          expect(metadata['single_object']['single_object_first_name']).to eq('Fake')
+          expect(metadata['single_object']['single_object_last_name']).to eq('Fakerson')
+          expect(metadata['single_object']['single_object_position']).to include('Leader', 'Jester', 'Queen')
+          expect(metadata['single_object']['single_object_language']).to eq('English')
         end
       end
 
       context 'with object fields and no prefix' do
         let(:importer) do
           FactoryBot.create(:bulkrax_importer_csv, field_mapping: {
-                              'first_name' => { from: ['creator_first_name'], object: 'creator' },
-                              'last_name' => { from: ['creator_last_name'], object: 'creator' },
-                              'position' => { from: ['creator_position'], object: 'creator' },
-                              'language' => { from: ['creator_language'], object: 'creator', parsed: true }
+                              'first_name' => { from: ['single_object_first_name'], object: 'single_object' },
+                              'last_name' => { from: ['single_object_last_name'], object: 'single_object' },
+                              'position' => { from: ['single_object_position'], object: 'single_object' },
+                              'language' => { from: ['single_object_language'], object: 'single_object', parsed: true }
+                            })
+        end
+
+        before do
+          allow_any_instance_of(ObjectFactory).to receive(:run!)
+          allow(subject).to receive(:record).and_return(
+            'source_identifier' => '2',
+            'title' => 'some title',
+            'single_object_first_name' => 'Fake',
+            'single_object_last_name' => 'Fakerson',
+            'single_object_position' => 'Leader, Jester, Queen',
+            'single_object_language' => 'english'
+          )
+        end
+
+        it 'succeeds' do
+          metadata = subject.build_metadata
+          expect(metadata['single_object']['first_name']).to eq('Fake')
+          expect(metadata['single_object']['last_name']).to eq('Fakerson')
+          expect(metadata['single_object']['position']).to include('Leader', 'Jester', 'Queen')
+          expect(metadata['single_object']['language']).to eq(['English'])
```

Comment:
@orangewolf although we set `single_object` to be `multiple:false`, this check now equates to an array of strings because the `language` property [here](https://github.com/samvera/hyrax/blob/b034218b89dde7df534e32d1e5ade9161e129a1d/app/models/concerns/hyrax/basic_metadata.rb) is defaulted to multiple.

`expect(metadata['single_object']['single_object_language']).to eq('English')` in the spec above still comes back as a string, since `single_object_language` is not a predefined property.


do we want to keep it this way? I [made an adjustment](https://github.com/samvera-labs/bulkrax/pull/289/commits/6aace7ef4f067c9d39820730042acb7d03f9bf49) that would alter `multiple?` to return false if the field OR the object were set to false. but I backtracked it because I started to doubt that direction.

Reply:
or would you prefer `expect(metadata['single_object']['single_object_language'])` be adjusted to also equal `['English']` since that goes along with the spirit of how `language` is supposed to react?

Reply:
this was resolved with the removal of `include ::Hyrax::BasicMetadata` from the "Work" class.